domain/application/modelmigration/import.go

```diff
@@ -671,14 +671,33 @@ func (i *importOperation) importCharmActions(data description.CharmActions) (*in
 
 func (i *importOperation) importEndpointBindings(app description.Application, spaces []description.Space) (map[string]network.SpaceName, error) {
 	endpointBindings := make(map[string]network.SpaceName)
+	// Get the application's default space
+	var applicationDefaultSpaceID string
+	for endpoint, spaceID := range app.EndpointBindings() {
+		if endpoint == "" {
+			applicationDefaultSpaceID = spaceID
```

Comment:
todo: add a `continue`, there is only one default space

Reply:
I've added a `break`, which I guess is what you meant :)